src/uct/sm/mm/base/mm_iface.h

```diff
@@ -249,6 +249,31 @@ typedef struct uct_mm_iface {
                   uct_mm_iface_config_t);
 
 
+/*
+ * Memory-mapper transport for MM registration routines
+ *
+ * @param _name         Component name token
+ * @param _md_ops       Memory domain operations, of type uct_mm_md_ops_t
+ * @param _rkey_unpack  Remote key unpack function
+ * @param _rkey_release Remote key release function
+ * @param _cfg_prefix   Prefix for configuration variables
+ * @param _cfg_table    Configuration table
+ */
+#define UCT_MM_TL_REGISTER_DEF(_name, _md_ops, _rkey_unpack, _rkey_release, \
```

Comment:
can we just update UCT_MM_TL_DEFINE definition? why need new macro?

Reply:
> since this PR changes all MM, why we still need the old macro?

yep, fixed